command/agent/alloc_endpoint_test.go

```diff
@@ -259,6 +260,24 @@ func TestHTTP_AllocSnapshot(t *testing.T) {
 	})
 }
 
+func TestHTTP_AllocSnapshot_WithMigrateToken(t *testing.T) {
```

Comment:
@dadgar if you have an idea how to access a client's secret identifier from this test, that would be helpful- I was considering creating a valid MigrateToken in the test and then querying for an allocation. Other ideas how to test the valid case would also be great.

Reply:
`s.Agent.Client().Node().SecretID`